lib/mixlib/install/generator/powershell/scripts/install_project.ps1

```diff
@@ -16,11 +16,6 @@ function Install-Project {
   [cmdletbinding(SupportsShouldProcess=$true)]
   param (
     # Project to install
-    # chef - Chef Client
-    # chefdk - Chef Development Kit
-    # angrychef - AngryChef
-    # server and container are not valid windows targets
-    [validateset('chef', 'chefdk', 'angrychef')]
     [string]
     $project = 'chef',
```

Comment:
With the other projects in mind, shouldn't the channels apply to them as well and not just `'chef'`?